src/lib/operators/abstract_operator.cpp

```diff
@@ -127,12 +127,9 @@ void AbstractOperator::execute() {
       const auto& lqp_expressions = lqp_node->output_expressions();
       if (!_output) {
         Assert(lqp_expressions.empty(), "Operator did not produce a result, but the LQP expects it to");
-      } else if (std::dynamic_pointer_cast<const AbstractNonQueryNode>(lqp_node) ||
```

Comment:
Why is the  `AbstractNonQueryNode` kicked out?

Reply:
Operators implementing behavior of an 'AbstractNonQueryNode' (i.e., `Insert/Update/Delete`, `Create/Drop[Table|View]`, `CreatePreparedPlan`, `Import/Export`, `ChangeMetaTable`) should not have an output (and `AbstractNonQueryNode::output_expressions()` returns an empty vector). Thus, they match the first `if` condition we do not have the exceptional case fo them anymore.